### ISSUE TITLE
Update

### ISSUE THREAD

**Issue by S-15-77** [S-15-77/S-15-77#1]
<h1 align="center">Hi 👋, I'm Santhosh Dayakar</h1>
<h3 align="center">Full-Stack Developer | AI Enthusiast | Cloud Explorer</h3>

<p align="center">
  <img src="https://readme-typing-svg.herokuapp.com?font=Fira+Code&pause=1000&color=40FFBF&width=435&lines=Welcome+to+my+hub!;Building+projects+that+solve+real+problems;Learning+something+new+every+day!" />
</p>

---

## 🧠 About Me

- 🔭 I’m currently working on **AI-powered apps with a focus on multimodal pipelines**

- 🌱 I’m deep-diving into **LLMs, WebSockets, Firebase, and cloud deployments**

- 💬 Ask me about **ML pipelines, Docker, React, Python, or how to launch products in a weekend**

- 🧪 Fun fact: I love debugging — it’s like solving little mysteries every day.

---

## 🔧 Languages and Tools

<div align="center">
  <img src="https://skillicons.dev/icons?i=python,java,js,ts,html,css,react,nextjs,nodejs,express,mongodb,mysql,postgres,firebase,aws,gcp,docker,git,kubernetes,tensorflow,pytorch,vscode,figma&theme=dark" />
</div>

---

## 📊 GitHub Stats

<p align="center">
  <img src="https://github-readme-stats.vercel.app/api?username=S-15-77&show_icons=true&theme=tokyonight&hide_border=false" width="47%" />
  <img src="https://github-readme-streak-stats.herokuapp.com/?user=S-15-77&theme=tokyonight&hide_border=false" width="47%" />
</p>
<p align="center">
  <img src="https://github-readme-stats.vercel.app/api/top-langs/?username=S-15-77&layout=compact&theme=tokyonight&hide_border=false" width="50%" />
</p>

---

## 🏆 Trophies

<p align="center">
  <img src="https://github-profile-trophy.vercel.app/?username=S-15-77&theme=gruvbox&no-frame=true&no-bg=true&margin-w=4" />
</p>

---

## ✍️ Quote of the Day

<p align="center">
  <img src="https://quotes-github-readme.vercel.app/api?type=horizontal&theme=dark" />
</p>

---

## 🌐 Connect With Me

[![LinkedIn](https://img.shields.io/badge/LinkedIn-%230077B5.svg?style=for-the-badge&logo=linkedin&logoColor=white)](https://linkedin.com/in/santhosh-dayakar-45aab6202/)
[![YouTube](https://img.shields.io/badge/YouTube-%23FF0000.svg?style=for-the-badge&logo=YouTube&logoColor=white)](https://www.youtube.com/channel/UCXzVuptoIlka0mSXp0GbaRw)
[![Instagram](https://img.shields.io/badge/Instagram-%23E4405F.svg?style=for-the-badge&logo=Instagram&logoColor=white)](https://www.instagram.com/sandiz.de/)
[![Portfolio](https://img.shields.io/badge/Portfolio-%2312100E.svg?style=for-the-badge&logo=firefox&logoColor=white)](https://santhoshdayakar.dev)

---

## ☕ Support Me

[![Buy Me A Coffee](https://img.shields.io/badge/Buy%20Me%20a%20Coffee-%23FFDD00.svg?style=for-the-badge&logo=buy-me-a-coffee&logoColor=black)](https://www.buymeacoffee.com/santhoshdaya)

---

<p align="center">
  <img src="https://komarev.com/ghpvc/?username=S-15-77&label=Profile+views&color=40ffbf&style=flat" />
</p>
